pmm/pmm2-ami-staging-start.groovy

```diff
@@ -151,6 +151,7 @@ pipeline {
                         if [ -n "$SSH_KEY" ]; then
                             echo '$SSH_KEY' | ssh -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@\$(cat IP_PUBLIC) 'cat - >> .ssh/authorized_keys'
                         fi
+                        [ ! -d "/home/centos" ] && echo "This Directory Doesn't Exist"
```

Comment:
Home directory for centos user does not exist.

Reply:
but this will be executed as a check when logged in as admin user